shared/chat/conversation/messages/cards/team-journey/container.tsx

```diff
@@ -53,7 +53,10 @@ const TeamJourneyContainer = (props: Props) => {
           : ['wave', {label: 'Publish team on your profile', onClick: props.onPublishTeam}]
       image = 'icon-illustration-welcome-96'
       textComponent = (
-        <Kb.Text type="BodySmall">Welcome to the team! Say hi to everyone and introduce yourself.</Kb.Text>
+        <Kb.Text type="BodySmall">
+          <Kb.Emoji allowFontScaling={true} size={16} emojiName=":wave:" /> Welcome to the team! Say hi to
```

Comment:
```suggestion
          <Kb.Emoji allowFontScaling={true} size={Styles.globalMargins.small} emojiName=":wave:" /> Welcome to the team! Say hi to
```
with `import * as Styles from '../../../../../styles'` at the top of the file